src/utils.rs

```diff
@@ -0,0 +1,117 @@
+use std::env;
+use std::ffi::OsString;
+use std::fs::File;
+use std::io::ErrorKind;
+use std::num::NonZeroI32;
+use std::path::Path;
+use std::process::{Command, Stdio};
+
+use pipe_trait::Pipe;
+
+use crate::error::{MainError, PnError};
+use crate::shell_quoted::ShellQuoted;
+use crate::NodeManifest;
```

Comment:
Let's not change how imports are organized.

```rust
use clap::Parser;
use cli::Cli;
use error::{MainError, PnError};
use indexmap::IndexMap;
use pipe_trait::Pipe;
use serde::{Deserialize, Serialize};
use shell_quoted::ShellQuoted;
use std::{
    env,
    ffi::OsString,
    fs::File,
    io::{self, ErrorKind, Write},
    num::NonZeroI32,
    io::{self, Write},
    path::Path,
    process::{exit, Command, Stdio},
    process::exit,
};
use yansi::Color::{Black, Red};
```

Reply:
I was using the `group_imports = "StdExternalCrate"` setting of rustfmt nightly

Reply:
Let's not change default.